### PR TITLE
[baxtereus] overwrite ros-state-callback in baxter-interface.l for su…

### PR DESCRIPTION
**Pull request by aginika**
…ppress torso warning

To prevent the warning of 
```
;; #<rotational-joint #X69c5998 torso_t0> :joint-angle(-719.978) violate min-angle(-172.46)
```

### CODE REVIEW

**Comment by aginika** [comment]
Could anyone merge this?

File: jsk_baxter_robot/baxtereus/baxter-interface.l [hunk 1]
```diff
@@ -246,6 +246,27 @@
 		   (setq tms (append tms (list 50))))
      (setq ctype (or ctype controller-type))  ;; use default if ctype is nil
 		 (send-super :angle-vector-sequence avs tms ctype start-time :scale scale :min-time min-time))
+  
+  (:ros-state-callback
+   (msg)
+   (let ((robot-state-names (cdr (assoc :name robot-state))) (robot-msg-names (send msg :name)) (torso-index))
+     ;;Remove toros_t0 . We think this is not the rotational-joint
+     (dotimes (i (length robot-msg-names))
+       (if (string= "torso_t0" (elt robot-msg-names i))
```
Comment (k-okada) [inline jsk_baxter_robot/baxtereus/baxter-interface.l]: You can use position

File: jsk_baxter_robot/baxtereus/baxter-interface.l [hunk 2]
```diff
@@ -246,6 +246,27 @@
...
+           (setq torso-index i)
+           )
+       )
+     (send msg :name (concatenate cons
```
Comment (k-okada) [inline jsk_baxter_robot/baxtereus/baxter-interface.l]: You can use remove

**Comment by aginika** [comment]
I updated. Could you check again?
I confirmed on real robot.

File: jsk_baxter_robot/baxtereus/baxter-interface.l [hunk 3]
```diff
@@ -246,6 +246,21 @@
 		   (setq tms (append tms (list 50))))
      (setq ctype (or ctype controller-type))  ;; use default if ctype is nil
 		 (send-super :angle-vector-sequence avs tms ctype start-time :scale scale :min-time min-time))
+  
+  (:ros-state-callback
+   (msg)
+   (let ((robot-state-names (cdr (assoc :name robot-state))) (robot-msg-names (send msg :name)) (torso-index))
+     ;;Remove toros_t0 . We think this is not the rotational-joint
+     (setq torso-index (position "torso_t0" robot-msg-names :test #'string=))
+     (send msg :name ( remove "torso_t0" robot-state-names :test #'string=))
```
Comment (k-okada) [inline jsk_baxter_robot/baxtereus/baxter-interface.l]: (when torso-index
  (send msg :name (remove ...))
  (dolist (key '(:....
)

may ok, position returns nil when it is not found and when 0 is true

**Comment by aginika** [comment]
I updated

**Comment by aginika** [comment]
Seems to be pass the test

**Comment by k-okada** [comment]
it should fail, I'll check this

**Comment by k-okada** [comment]
move to #338 with test

**Pull request closed without merging**